polaris-react/src/components/Button/Button.stories.tsx

```diff
@@ -844,26 +847,45 @@ export function CopyToClipboard() {
     defaultValue: 'hello@example.com',
   });
 
+  const ref = useRef(null);
+  const isFocusedIn = useFocusIn(ref);
+  const isHovered = useHover(ref);
+  const isMouseDevice = useMediaQuery('mouse');
+  const isMouseHovered = isMouseDevice ? isHovered : true;
```

Comment:
Very nice! This pattern feels much more composable to me. Thanks for updating this so fast ⚡